Navaids/VOR_Non-UK.txt

```diff
@@ -53,23 +53,33 @@ REM 112.300 N049.18.41.900 E004.02.43.300 ; Reims
 REN 109.250 N048.04.09.300 W001.44.30.000 ; Rennes Saint Jacques
 ROU 116.800 N049.27.56.300 E001.16.50.300 ; Rouen Vallee de Seine
 
-;Ireland, verified 2012/01
+;Ireland, verified AIRAC 2308
 BAL 115.800 N053.17.59.600 W006.26.52.000 ; Baldonnel
 CON 117.400 N053.54.28.900 W008.49.12.400 ; Connaught
 CRK 114.600 N051.50.26.200 W008.29.39.400 ; Cork
 DAP 111.200 N053.25.25.200 W006.18.10.300 ; Collinstown
 DUB 114.900 N053.29.57.800 W006.18.25.600 ; Dublin
 ;GTG 114.150 N052.51.11.100 W006.08.43.900 ; Gleinteig (DME)
```

Comment:
Not sure why this is commented? Still seems active. Maybe as DME only? Which makes me wonder if we want to define Non-UK DME Only stations?

Reply:
Yep seems still active, I just left it as it was to be honest, I can add it back in if you think that's good.